tests/helpers/drupal-commands.js

```diff
@@ -7,11 +7,17 @@ module.exports = {
    * Finds node ID via drush and visits node edit page.
    * @param  {Array<{page: Page, node_title: String}>} array Page object and
    *   node title
+   * @param {string} langcode Optional language code prefix for the URL.
+   *   Note the node title must be in the original language. The visited page
+   *   will edit the node's translation if it exists, otherwise the original
+   *   language node (just in another UI language).
    * @return {Response} The response.
    */
-  visitNodeEditPage: async ([page, node_title]) => {
+  visitNodeEditPage: async ([page, node_title], langcode = '') => {
```

Comment:
This is the first time that we are adding a second/optional function parameter here.

TBH I don't know why all function parameters here are arrays instead of separate parameters. I didn't see a convenient way to implement this, besides adding a separate parameter (which is not going to be used very often).